Makefile

```diff
@@ -1,9 +1,10 @@
-GOFLAGS ?= "-tags=${CLOUD_PROVIDER}"
-
 RELEASE_REPO ?= public.ecr.aws/b6u6q9h4
-RELEASE_VERSION ?= v0.2.0
+RELEASE_VERSION ?= $(shell git describe --tags --always --dirty)
 RELEASE_MANIFEST = releases/${CLOUD_PROVIDER}/manifest.yaml
 
+## Inject the app version into project.Version
+LDFLAGS ?= "-ldflags=-X=github.com/awslabs/karpenter/pkg/utils/project.Version=${RELEASE_VERSION}"
```

Comment:
How do these get picked up. Don't we need something on line 8?

Reply:
No, line 8 already includes ${GOFLAGS} which includes the LDFLAGS since they are apart of the GOFLAGS.

Reply:
![Uploading image.png…]()